dashboard/config/base.webpack.config.js

```diff
@@ -117,16 +120,23 @@ export default ({
             ],
           },
           {
-            test: /\.(js|jsx)$/,
-            exclude: path.resolve(root, "node_modules"),
+            test: /\.(mjs|js|jsx)$/,
+            include: [
+              path.join(root, "src"),
+              path.join(root, "node_modules/@sensu/web-ui/src"),
+            ],
             loader: require.resolve("babel-loader"),
             options: {
+              babelrcRoots: [
+                root,
+                path.join(root, "node_modules/@sensu/web-ui"),
```

Comment:
I can't find where this is consumed? Is this so that can be consumed by babel on the other side?

Reply:
How would you feel about `@sensuapp/web`? We own the `@sensuapp` npm org. Eventually we could have `@sensuapp/sdk`.

Reply:
Yeah, this is for when the config is reused elsewhere